### ISSUE TITLE
Typo dict_1.match/dict_1.match_to_reference breaks build from source

### ISSUE THREAD

**Issue by balducci** [pemsley/coot#173]
hi there

while building from source latest master (11c61eed45fd2b1b16a8f5646a04eccf9e1a3b9b) I am getting:

```
    <molecules-container.cc: In member function 'std::map<std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> > molecules_container_t::dictionary_atom_name_map(const std::string&, int, const std::string&, int)':
    molecules-container.cc:5828:52: error: 'const class coot::dictionary_residue_restraints_t' has no member named 'match'
     5828 |          coot::dictionary_match_info_t dm = dict_1.match(dict_2, nullptr, comp_id_1, "dummy");
          |                                                    ^~~~~
    make[2]: *** [Makefile:801: molecules-container.lo] Error 1
    make[2]: *** Waiting for unfinished jobs....

```

From browsing other sources it seems to me that ``dict_1.match`` should actually be ``dict_1.match_to_reference``, as per ``protein-geometry.hh`` ?

ciao

-gabriele

**Comment by pemsley** [comment]
More code accidentally committed.

**Comment by pemsley** [comment]
Fixed in 9e5d80f31901a255ea752489b1d8f8576fd23f49.
